charts/ingress-nginx/templates/default-backend-poddisruptionbudget.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.defaultBackend.enabled -}}
 {{- if or (gt (.Values.defaultBackend.replicaCount | int) 1) (gt (.Values.defaultBackend.autoscaling.minReplicas | int) 1) }}
 apiVersion: policy/v1beta1
```

Comment:
Let's verify what's the cluster version here.

See:
https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/controller-poddisruptionbudget.yaml#L2

Reply:
I think that I did the wrong thing, rebasing from main branch.
Sorry for the inconvenience.
It this is a problem, I will close the PR.